src/Service.Tests/SqlTests/RestApiTests/Patch/PatchApiTestBase.cs

```diff
@@ -589,12 +589,14 @@ public virtual async Task PatchWithNoPrimaryKeyRouteTest()
         }
 
         /// <summary>
-        /// Test to validate that PATCH operation on inaccessible row (by virtue of database policy) fails.
+        /// Test to validate that PATCH operation which does not satisfy the database policy ("@item.id ne 1234")
+        /// has no rows accessible to update and thus fails.
```

Comment:
```suggestion
        /// Test to validate that PATCH operation fails because the database policy ("@item.id ne 1234")
        /// restricts viewing and modifying records where id is not 1234.
```
you can also remove  `/// <returns></returns>` right below this

Reply:
Not exactly viewing records, but modifying records, yes.